packages/account-v2/src/constants/tradingAssessmentQuestions.ts

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+export const getTradingAssessmentQuestions = () => [
+    {
+        answerOptions: [
```

Comment:
should we move this to modules so that real account creation can use the same constants file

Reply:
will move it to modules 🙏